Mathlib/Topology/MetricSpace/Isometry.lean

```diff
@@ -483,6 +484,39 @@ theorem completeSpace_iff (e : α ≃ᵢ β) : CompleteSpace α ↔ CompleteSpac
 protected theorem completeSpace [CompleteSpace β] (e : α ≃ᵢ β) : CompleteSpace α :=
   e.completeSpace_iff.2 ‹_›
 
+/-- `Equiv.piCongrLeft` as an `IsometryEquiv`: this is the natural
+`∀ i, Y i ≃ᵢ ∀ j, Y (e.symm j)` obtained from a bijection `ι ≃ ι'` of finite types.-/
+def piCongrLeftofFintype' {ι' : Type*} [Fintype ι] [Fintype ι'] {Y : ι → Type*}
```

Comment:
I think you can probably drop `ofFintype` from the name here and below, since the metric space structure is present in the instance for metric spaces on pi types